index.html

```diff
@@ -0,0 +1,571 @@
+
+## The Basics
+
+Core schemas:
+  1. **must** be valid GraphQL schema documents,
+  2. **must** contain exactly one `SchemaDefinition`, and
+  3. **must** use the `@core` directive on their schema definition to declare any features they reference by using [`@core(feature:)`](#core) to reference a [well-formed feature URL](#feature-urls).
+
+The first `@core` directive on the schema **must** reference the core spec itself, i.e. this document.
+
+<view-of src=basic.graphql title='Basic core schema using @core and @example'></view-of>
+
+### Unspecified directives are passed through
+
+Existing schemas likely contain definitions for directives which are not versioned, have no specification document, and are intended mainly to be passed through. This is the default behavior for core schema processors:
+
+```graphql
+#::[Unspecified directives are passed through]
+schema
+  @core(feature: "https://lib.apollo.dev/core/v0.1")
+{
+  query: Query
+}
+
+type SomeType {
+  field: Int @another
+}
+
+# `@another` is unspecified. Core processors will not extract metadata from
+# it, but its definition and all usages within the schema will be exposed
+# in the export schema.
+directive @another on FIELD
+```
+
+### Renaming `@core` itself
+
+It is possible to rename the `@core` directive itself with the same `as:` mechanism used for all features:
+
+```graphql
+#::[Renaming `@core` to `@coreSchema`]
+schema
+  @coreSchema(feature: "https://lib.apollo.dev/core/v0.1", as: "coreSchema")
+  @coreSchema(feature: "https://example.com/example/v1.0")
+{
+  query: Query
+}
+
+type SomeType {
+  field: Int @example
+}
+
+directive @example on FIELD
+```
+
+## `@core`
+
+```graphql
+#::[The `@core` directive]
+directive @core(feature: core__FeatureUrl!, as: String, export: Boolean)
+  repeatable on SCHEMA
+```
+
+Declare a core feature present in this schema.
+
+Documents **must** include a definition for the `@core` directive. The provided definition must be *compatible* with the definition above, but may:
+- **Omit optional arguments** if they are never used in the document,
+- **Omit locations** where the directive never occurs,
+- **Introduce new directive locations**, the behavior of which is unspecified here.
+- **Introduce new arguments.** New arguments **must** be prefixed, e.g. `example__extensionArgument: Bool`. The prefix **must** be the name of a feature referenced with this or another `@core` directive within the document.
+- **Use `String` in place of custom scalars.** This is an ease-of-authoring affordance which allows document authors to omit the definitions of custom scalar types provided by features. Processors **must** continue to interpret such arguments and fields according to the encoding and decoding rules of the custom scalar type. For example, the `@core` directive may be defined as follows, with no change to the behavior:
```

Comment:
In this case, it literally just means the example below (String instead of core__FeatureUrl), so this can be written less generically as saying you can use `String` instead of `core__FeatureUrl`.

However, my advice would be that we should just go back to using `String` instead of `core__FeatureUrl`. I don't think that much is added to this spec by declaring a custom scalar type for this argument which needs to be declared by users every time they use `@core`... except that you basically say here that you don't have to do it anyway.  We chatted on Slack a bit and if I may paraphrase, the idea is that it makes the implementation in the core-schema-js/core-schema-rust libraries a little easier, but I think the number of schemas that will need the extra `scalar core__FeatureUrl` line is far greater than the number of libraries that will be written to parse `@core` directives, and at least in the TS version this is going in custom bootstrap code anyway! I'll also note that the standard `@specifiedBy` directive's argument is just a String.  If you're going to make this concession that most users will be lazy and just write `String` anyway, I think it would be simplest to just have it be a `String`, and document the structure of these URLs just in the `feature:` section.